scripts/bumpChangelogs/getChangelogCommitMessage.js

```diff
@@ -0,0 +1,9 @@
+module.exports = packageNames => {
+  const hasMultiplePackages = packageNames.length > 1;
+  const headline = `Update changelog${
```

Comment:
two `hasMultiplePackages` checks seem unnecessary here, as does `Update changelog{s}` and `Updates changelogs`, can these be combined? Seems like were doing the same logic twice, in opersite ways, and joining them

Reply:
@dr3 How's the code for that looking now?

Reply:
Yeah thats nicer :)